xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/HuobiCurrencyWrapper.java

```diff
@@ -0,0 +1,53 @@
+package org.knowm.xchange.huobi.dto.marketdata;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Arrays;
+
+public class HuobiCurrencyWrapper {
+
+    private HuobiCurrency[] huobiCurrencies;
+    private String currency;
+    private String instStatus;
+
+    public HuobiCurrencyWrapper(@JsonProperty("chains") HuobiCurrency[] huobiCurrencies,
+                                @JsonProperty("currency") String currency,
+                                @JsonProperty("instStatus") String instStatus) {
+        this.huobiCurrencies = huobiCurrencies;
+        this.currency = currency;
+        this.instStatus = instStatus;
+    }
+
+    public HuobiCurrency[] getHuobiCurrencies() {
+        return huobiCurrencies;
+    }
+
+    public void setHuobiCurrencies(HuobiCurrency[] huobiCurrencies) {
+        this.huobiCurrencies = huobiCurrencies;
+    }
```

Comment:
we still have setters in this DTO which are not needed

Reply:
@walec51 ups, I missed that. Fixed now